### ISSUE TITLE
pretrained weights

### ISSUE THREAD

**Issue by WxlSky** [woshisad159/TFNet#5]
Great work! Will the model weights be made publicly available? Could I request access to the trained weights?

**Comment by woshisad159** [comment]
If you need then I can upload the trained weights.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "woshisad159/TFNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年2月21日(星期五) 下午5:19
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[woshisad159/TFNet] pretrained weights (Issue #5)



   
Great work! Will the model weights be made publicly available? Could I request access to the trained weights?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;
  WxlSky created an issue (woshisad159/TFNet#5)
 
Great work! Will the model weights be made publicly available? Could I request access to the trained weights?

**Comment by WxlSky** [comment]
Yes, I would like to build upon your work for further research. Thank you for open-sourcing the model weights

**Comment by woshisad159** [comment]
Ok, I have put the latest trained model weights address in the README, the test set in the CE-CSL dataset can reach 32.46%, the processing method does not need to change the model, just need to pre-train on CSLDaily first, there is a small change in Net.py, remember to update it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "woshisad159/TFNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年2月22日(星期六) 中午12:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [woshisad159/TFNet] pretrained weights (Issue #5)



   
Yes, I would like to build upon your work for further research. Thank you for open-sourcing the model weights

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;
  WxlSky left a comment (woshisad159/TFNet#5)
 
Yes, I would like to build upon your work for further research. Thank you for open-sourcing the model weights

**Comment by WxlSky** [comment]
great! thank you for open-sourcing the model weights.
